cmd/create/cluster/cmd.go

```diff
@@ -2345,12 +2366,20 @@ func handleByoOidcOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool) (bool
 	return isByoOidcSet, oidcEndpointUrl, oidcPrivateKeySecretArn
 }
 
-func minReplicaValidator(multiAZ bool) interactive.Validator {
+func minReplicaValidator(multiAZ bool, isHostedCP bool, privateSubnetsCount int) interactive.Validator {
 	return func(val interface{}) error {
 		minReplicas, err := strconv.Atoi(fmt.Sprintf("%v", val))
 		if err != nil {
 			return err
 		}
+		if isHostedCP {
+			if minReplicas%privateSubnetsCount != 0 {
+				return fmt.Errorf("Hosted clusters require that the number of compute nodes be a multiple of "+
+					"the number of private subnets %d, instead received: %d", privateSubnetsCount, minReplicas)
```

Comment:
Q: is `privateSubnetsCount` used here as a proxy for the number of AZs, assuming one private subnet per AZ?  
Or can HostedCP clusters have multiple private subnets in same AZ, and you meant this literally?